### DISCUSSION TITLE
Can't list organization projects (beta) with GraphQL API

### DISCUSSION THREAD

**Discussion by ElMatella** [community/community#19802] [Projects and Issues]
I can't list my organization projects. 

However, I can read a project details if I provide a project number:

  

    await octokit.graphql(`
        query($organization: String!){
          organization(login: $organization){
            projectV2(number: 1) {
              id
              title
            }
          }
        }
      `, {
        organization: 'xxx'
      })

This works as expected and returns a valid project. However:

  

    await octokit.graphql(`
        query($organization: String!){
          organization(login: $organization){
            projectsV2(first: 20) {
              nodes {
                id
                title
              }
            }
          }
        }
      `, {
        organization: 'xxx'
      })

This returns an empty array.

What could I possibly miss? I have admin rights on projects with this Github APP.

Here is the documentation I'm following: https://docs.github.com/en/issues/trying-out-the-new-projects-experience/using-the-api-to-manage-projects#finding-the-node-id-of-an-organization-project

The same request works well with a user token, but not with an App token

**Comment by nickserv** [comment]
I have the same issue using the v3 API. The docs mention that GitHub Apps aren't supported, but I don't understand why. It works with an OAuth App. Because of this, I have to develop a new GitHub integration using an OAuth App instead of a recommended GitHub App.

**Comment by nairihar** [comment]
<img width="882" alt="Screen Shot 2023-02-01 at 12 54 15 PM" src="https://user-images.githubusercontent.com/15065065/216035698-3c482ed3-8d8b-4855-9cb7-56fa2247cb68.png">
